src/alt/store/StoreMixin.js

```diff
@@ -38,9 +38,10 @@ const StoreMixin = {
       publicMethods[methodName] = (...args) => {
         const state = this.getInstance().getState()
         const value = asyncSpec.local && asyncSpec.local(state, ...args)
+        const shouldFetch = typeof asyncSpec.shouldFetch === 'function' ? asyncSpec.shouldFetch(state, ...args) : !value
```

Comment:
You mean if shouldFetch: false. 

Why not just always make it a function?

shouldFetch: () => false